types/oauth2-server/index.d.ts

```diff
@@ -381,6 +382,37 @@ declare namespace OAuth2Server {
         user: User;
         [key: string]: any;
     }
+
+    /**
+     * Error classes
+     * The base class is OAuthError
+     */
+    class OAuthError extends Error {
```

Comment:
Unless I'm reading the [docs](https://oauth2-server.readthedocs.io/en/latest/api/errors/invalid-scope-error.html) or [implementation](https://github.com/oauthjs/node-oauth2-server/blob/master/lib/server.js) wrong, the root OAuth2 server doesn't actually export members for these errors available at runtime - to access them and `new` them up, you need to require them directly - the declaration file(s) need to be organized in the same way, since what's here might typecheck, but will fail at runtime (the other members are fine because they're interfaces and have no runtime part).

Reply:
Yes, you're correct.  My bad there :). Closing pr.